css/login.css

```diff
@@ -0,0 +1,164 @@
+body{
+	background-color: #555555;
+	margin: 0;
+	font-family: 'Roboto', sans-serif;
+	min-width: 1200px;
+}
+
+.rectangle{
+	width: 100%;
+	height: 640px;
+}
+
+.firstRectangle{
+	background-image: url('../img/feed.jpg');
+	background-blend-mode: overlay;
+}
+
+.whiteRectangle{
+	width: inherit;
+	height: inherit;
+	background-color: #CBF6FF;
+	opacity: 0.3;
+	position: absolute;
+}
+
+.logInBox{
+	width: 375px;
+	height: 400px;
+	margin-left: 400px;
+	background-color: #F5F5F5;
+	opacity: 1!important;
+	position: absolute;
+	margin-top: 140px;
+	box-shadow: 0px 0px 8px black;
+	border-radius: 3px;
+}
+
+.headLogIn{
+	width: inherit;
+	background-color: #1ab2ff;
+	height: 75px;
+	box-shadow: 0px 3px 10px grey;
+	display: table;
+}
+
+.headLogIn > div{
```

Comment:
Why first-level `div`s only? What other types of elements does `.headLogIn` contain?